tests/arch/efinix/mux.ys

```diff
@@ -36,6 +36,6 @@ proc
 equiv_opt -assert -map +/efinix/cells_sim.v synth_efinix # equivalency check
 design -load postopt # load the post-opt design (otherwise equiv_opt loads the pre-opt design)
 cd mux16 # Constrain all select calls below inside the top module
-select -assert-count 12 t:EFX_LUT4
+select -assert-min 11 t:EFX_LUT4
```

Comment:
Wouldn't it be better to do `-assert-max`?

Reply:
Was thinking more :
select -assert-min 11 t:EFX_LUT4
select -assert-max 12 t:EFX_LUT4
That will be then just covering two possible cases

Reply:
Use -assert-max 12 (as discussed)

Reply:
Updated